pkg/kgo/source.go

```diff
@@ -1041,6 +1041,7 @@ func (s *source) handleReqResp(br *broker, req *fetchRequest, resp *kmsg.FetchRe
 
 		fetchTopic := FetchTopic{
 			Topic:      topic,
+			TopicID:    rt.TopicID,
```

Comment:
Nice, so if version >= 13, I should be assured that this is set?

Reply:
It's set if the broker returns it -- it's a new field in fetch response v13, yes -- and this will be all 0s if it is unset.

Reply:
Gotcha, I assume all 0s is an invalid topic id.